src/wallet/feebumper.h

```diff
@@ -23,34 +21,32 @@ enum class BumpFeeResult
     MISC_ERROR,
 };
 
-/* return whether transaction can be bumped */
-bool TransactionCanBeBumped(CWallet* pWallet, const uint256& txid);
+//! Return whether transaction can be bumped.
+bool TransactionCanBeBumped(CWallet* wallet, const uint256& txid);
 
-/* create bumpfee transaction */
-BumpFeeResult CreateBumpFeeTransaction(const CWallet* pWallet,
+//! Create bumpfee transaction.
+BumpFeeResult CreateBumpFeeTransaction(const CWallet* wallet,
     const uint256& txid,
     const CCoinControl& coin_control,
-    CAmount totalFee,
-    std::vector<std::string>& vErrors,
-    CAmount& nOldFee,
-    CAmount& nNewFee,
+    CAmount total_fee,
```

Comment:
Yes, please do this, but also please keep names consistent between the cpp and h.